experiment/simulation/js/anim.js

```diff
@@ -0,0 +1,484 @@
+import { setCoordinates,fillInputDots,fillColor,objectDisappear,objectAppear,setter, calculateNot} from "./animation-utility.js";
+
+'use strict'
+
+window.appendSI1 = appendSI1;
+window.appendSI2 = appendSI2;
+window.appendSI3 = appendSI3;
+window.appendSI4 = appendSI4;
+window.appendClock = appendClock;
+window.simulationStatus = simulationStatus;
+window.restartCircuit = restartCircuit;
+window.setSpeed=setSpeed;
+
+let stage_num = 1;
+
+
+
+// Dimensions of working area
+const circuitBoard = document.getElementById("circuit-board");
+const sidePanels = document.getElementsByClassName("v-datalist-container");
+// Distance of working area from top
+const circuitBoardTop = circuitBoard.offsetTop;
+// Full height of window
+const windowHeight = window.innerHeight;
+const width = window.innerWidth;
+
+const svg = document.querySelector(".svg");
+const svgns = "http://www.w3.org/2000/svg";
```

Comment:
This link will probably give errors if you change it to https, in that case we can keep it to http till we are able to fix the errors.